source/users.rst

```diff
@@ -0,0 +1,130 @@
+
+A user can both consume data and produce data, or just have one of these roles (i.e., at the start/end of the production chain).
+
+
+Producers
+=========
+
+
+=========  ================================================  ================================================  ================================================
+  Level                       Automatic                                            Manual                                              Human
+=========  ================================================  ================================================  ================================================
+    1      * Weather station observation data                * Manual weather observations at a station
+           * Car, bus, self driving vehicle data             * Crowdsourced observations                       * Crowdsourced observations
+           * Vegvesenet roadside sensors, webcameras         * Biodiversity data (NINA) - manual               * Biodiversity data (NINA) - fieldwork 
+           * Private weather station data (Netatmo)
+           * Boat, buoy, oil platform observation data
+           * Drone, gliders (underwater) etc
+           * Radiosonde data
+           * Biodiversity data (NINA) - automatic
+           * Radar and HF radar data
+           * Satellite data
+           * Lightning sensor data
+    2      * Quality control system                          * Human quality control - manual control          * Human quality control - subjective control
+           * Aggregated time series                          * Producer manually creates aggregated dataset    * Producer subjectively creates a dataset 
+           * Lightning strike location data                  * Produce manually creates calibrated dataset
+           * Calibrated products based on radar data         * Producer manually creates merge dataset 
+           * Calibrated products based on satellite data
+           * Merged products based on level 1 data
+    3      * Resampled time series                           * Producer manually resamples time series
+           * Gridded datasets based on level 1 data
+    4      * Weather model data                              * Warnings forecast                               * Climate predictions
+           * Ocean model data                                * Avalanche forecast
+           * Air quality model                               * Nowcast
+           * Climate models                                   
+=========  ================================================  ================================================  ================================================
+
+**In situ observations:**
```

Comment:
Line 47 to 60: I'm not sure what you want to say here... Maybe it can be removed?

Reply:
I thought maybe those comments were still relevant? but I suppose they can be deleted if you don't think they are needed.

Reply:
Ok, changed my mind and deleted them...

Reply:
Ok :) (I've been to the optician, and now lunch..)